portal/common/lib/page_fetching.ts

```diff
@@ -17,7 +17,18 @@ import { aggregatorEndpoint } from "./aggregator";
 export async function resolveAndFetchPage(parsedUrl: DomainDetails): Promise<Response> {
     const rpcUrl = getFullnodeUrl(NETWORK);
     const client = new SuiClient({ url: rpcUrl });
+    const objectId = await resolveObjectId(parsedUrl, client);
+    if (typeof objectId == "string") {
```

Comment:
Question: I think we lose the `fullNodeFail()` case, right?

Reply:
Good catch @giac-mysten ! I just patched it